src/azure-cli/azure/cli/command_modules/acr/_utils.py

```diff
@@ -328,17 +328,15 @@ def get_custom_registry_credentials(cmd,
 
         custom_registries = {login_server: custom_reg_credential}
 
-    Credentials = cmd.get_models('Credentials')
     return Credentials(
         source_registry=source_registry_credentials,
         custom_registries=custom_registries
     )
 
 
 def build_timers_info(cmd, schedules):
-    TimerTrigger, TriggerStatus = cmd.get_models(
-        'TimerTrigger', 'TriggerStatus')
     timer_triggers = []
+    TriggerStatus, TimerTrigger = cmd.get_models('TriggerStatus', 'TimerTrigger', operation_group='tasks')
```

Comment:
when operation_group is empty it will try to load from the general sdk right? 2020 in this case

Reply:
When operation_group is empty it will load the default version `2020-11-01-preview`.
It has been defined in `_shared.py`.
For `tasks`, we need `2019-06-01-preview`